### ISSUE TITLE
Notify user on start up if config file fails to load

### ISSUE THREAD

**Issue by Silicasandwhich** [leftwm/leftwm#1104]
### Describe the feature

Especially with the new layout format in the newest git commits, it would be nice to somehow tell the user if their config file failed to load. This could take the form of just a terminal window with leftwm-check on it, and something discussed in the discord is the ability to define a custom script to load in this instance as well. Also, it would be nice if there was an option to open up $EDITOR in this instance as well since the user's keybindings won't be loaded.

### How important is this feature to you?

Would be nice to have

### How important do you think is this feature for the average LeftWM user?

Would probably be nice to have

**Comment by Silicasandwhich** [comment]
so ive done a little testing here, and it appears that the best way to do this is add another binary (called leftwm-panic or something) that gets called when the config fails to parse. since ive got another pull request currently open that adds yet another binary, im trying to find other ways to do this, but the more i look into other options, the more that occam's razor stares back at me

**Comment by Eskaan** [comment]
Possibly linked: #1105

**Comment by VuiMuich** [comment]
How about `leftwm-check --panic`?

**Comment by Silicasandwhich** [comment]
yeah, im realizing now that a flag on leftwm-check makes more sense. now im trying to get spawning the terminal window working properly, so im looking at i3 and seeing how they do it

**Comment by VuiMuich** [comment]
Well very basic command execution would be like this I guess:
```rust
std::process::Command::new("leftwm-check".to_string()).args("--panic".to_string()).spawn()?;
```

**Comment by Silicasandwhich** [comment]
yeah, that'll spawn the process, but it has to actually display inside a terminal window for it to actually be of use to the user. thankfully, it appears that every terminal in the default terminal vector supports passing commands using the `-e` flag (besides kitty and gnome, which just treat every argument after the flags as a command), so right now I'm trying to work out why launching alacritty with leftwm-check -p (which is currently set up to block until receiving input) when the config fails to parse does nothing, which i hoped i could find some clues for in the source code of i3, but that doesn't appear to be the case

**Comment by Silicasandwhich** [comment]
so ive come back to this after a bit of being too busy to really work on it, and while i don't have a solution quite yet, my hypothesis is that the problem is exactly when the terminal is getting launched, which is the second the config fails to load. at this point, it's possible that the wm isn't fully set up yet, so alacritty's not sure what to do. the main problem with this theory is that if this was the case, soft reloading a broken config should launch a terminal, but it doesn't. my other theory is that the `process::Command` is malformed, but considering that it's copy and pasted verbatim from a working test program, im not confident in that assertion.
